src/template/day.rs

```diff
@@ -44,7 +47,8 @@ impl Day {
 impl Day {
     /// Returns the current day if it's between the 1st and the 25th of december, `None` otherwise.
     pub fn today() -> Option<Self> {
-        let today = Local::now();
+        let offset = FixedOffset::east_opt(SERVER_UTC_OFFSET * 3600)?;
```

Comment:
curious why this uses `east_opt(-5 * 3600)` istd. of `west_opt(5 * 3600)`? (https://docs.rs/chrono/latest/chrono/offset/struct.FixedOffset.html#method.west_opt)

Reply:
The server time zone is UTC−5; I felt it was more clear to keep the sign as part of the offset.